<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janela_entrar_competicao</class>
 <widget class="QMainWindow" name="janela_entrar_competicao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background: #4A90E2;
}

QLabel{
	font-family: &quot;Segoe UI&quot;;
	font-weight: bold;
	font-size: 20px;
    color: #fff;
}

#_txt_titulo{
	font-size: 26px;
}

QPushButton{
    background: #4A90E2;
    border: 2px solid #fff;
    border-radius: 12px;
    padding: 10px 20px;

	font-family: &quot;Segoe UI&quot;;
	font-weight: bold;
	font-size: 16px;
    color: #fff;
}

QPushButton:hover{
    background: #53A3FF;
    border: 2px solid #fff;
    border-radius: 12px;
    padding: 10px 20px;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="_txt_titulo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Competição:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_iniciante">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciante</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_amador">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Amador</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_intermedirio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Intermediário</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_avancado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Avançado</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_iniciante">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Participar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_amador">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Participar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_intermediario">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Participar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_avancado">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Participar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_voltar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>396</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
